README.md

```diff
@@ -1 +1,2 @@
-# Ansible_playbooks
\ No newline at end of file
+# Ansible_playbooks
+Sample ansible command to run playbook ansible-playbook -v -i inventory.yml playbook.yml -e "nexus_dir=/home/projectdec2021/nexus nexus_server_ip=34.123.62.189 new_password=priya nexus_download_url=https://download.sonatype.com/nexus/3/latest-unix.tar.gz"
```

Comment:
I dont want any parameters to be sent from command line, all variables should be part of vars folder

Reply:
I have kept all the variables in vars folder